actors/util/adt/map.go

```diff
@@ -86,34 +92,42 @@ func (m *Map) Put(k abi.Keyer, v cbor.Marshaler) error {
 	return nil
 }
 
-// Get puts the value at `k` into `out`.
+// Get retrieves the value at `k` into `out`, if the `k` is present and `out` is non-nil.
+// Returns whether the key was found.
 func (m *Map) Get(k abi.Keyer, out cbor.Unmarshaler) (bool, error) {
-	if err := m.root.Find(m.store.Context(), k.Key(), out); err != nil {
-		if err == hamt.ErrNotFound {
-			return false, nil
-		}
+	if found, err := m.root.Find(m.store.Context(), k.Key(), out); err != nil {
 		return false, errors.Wrapf(err, "map get failed find in node %v with key %v", m.lastCid, k.Key())
```

Comment:
nit: we can use this as an excuse to switch to `xerrors.Errorf` if you want.

Reply:
Done, there was only one left untouched after this.